### PR TITLE
LPS-133450 | master

### PR DESCRIPTION
**Pull request by ThalitaCeli**

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/masterspagetemplate/MastersPageTemplates.testcase [hunk 1]
```diff
@@ -1710,4 +1710,62 @@ definition {
 		}
 	}
 
+	@description = "This is for LPS-133450. View the master page reverts to default. Test reported in bug LPS-131982."
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/masterspagetemplate/MastersPageTemplates.testcase:1713]: Should be This is a test for LPS-131982. Change master of content page with HTML title.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/masterspagetemplate/MastersPageTemplates.testcase [hunk 2]
```diff
@@ -1710,4 +1710,62 @@ definition {
...
+	@priority = "3"
+	test ViewMastersPageReturnToDefault {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/masterspagetemplate/MastersPageTemplates.testcase:1715]: Should be ChangeMasterOfContentPageWithHTMLTitle

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/masterspagetemplate/MastersPageTemplates.testcase [hunk 3]
```diff
@@ -1710,4 +1710,62 @@ definition {
...
+			Button.click(button = "Publish Master");
+
+			SitePages.openPagesAdmin(siteURLKey = "test-site-name");
+
+			ContentPages.addPage(
+				masterLayoutName = "Custom Masters Page Template",
+				pageName = "Test Page Name");
+		}
+
+		task ("Assert in the Master tab if the Custom Masters Page Template is selected") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/masterspagetemplate/MastersPageTemplates.testcase:1728]: The description is different with steps.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/masterspagetemplate/MastersPageTemplates.testcase [hunk 4]
```diff
@@ -1710,4 +1710,62 @@ definition {
...
+			PageEditor.gotoTab(tabName = "Page Design Options");
+
+			Click(locator1 = "PageEditor#MORE_LINK");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/masterspagetemplate/MastersPageTemplates.testcase:1731]: Should be
`		AssertElementPresent(
			key_card = "${masterLayoutName}",
			locator1 = "Card#CARD_SELECTED");
`

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/masterspagetemplate/MastersPageTemplates.testcase [hunk 5]
```diff
@@ -1710,4 +1710,62 @@ definition {
...
+				menuItem = "Configure",
+				pageName = "Test Page Name");
+
+			Navigator.gotoNavTab(navTab = "SEO");
+
+			PortletEntry.inputText(
+				fieldLabel = "HTML Title",
+				text = "Page HTML Title");
+
+			Button.clickSave();
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/masterspagetemplate/MastersPageTemplates.testcase:1752]: Should use PortletEntry.save();

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/masterspagetemplate/MastersPageTemplates.testcase [hunk 6]
```diff
@@ -1710,4 +1710,62 @@ definition {
...
+			Click(locator1 = "PageEditor#MORE_LINK");
+
+			AssertTextEquals(
+				key_masterLayoutName = "Custom Masters Page Template",
+				locator1 = "SitePagesEditPage#MASTER_LAYOUT_NAME",
+				value1 = "Custom Masters Page Template");
+		}
+
+		task ("Configure the HTML Title field") {
+			SitePages.openPagesAdmin(siteURLKey = "test-site-name");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/masterspagetemplate/MastersPageTemplates.testcase:1740]: Should access to configuration via page editor not Pages admin

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/masterspagetemplate/MastersPageTemplates.testcase [hunk 7]
```diff
@@ -1710,4 +1710,62 @@ definition {
...
+
+			PortletEntry.inputText(
+				fieldLabel = "HTML Title",
+				text = "Page HTML Title");
+
+			Button.clickSave();
+		}
+
+		task ("Assert in the Master tab if the Custom Masters Page Template is selected") {
+			ContentPagesNavigator.openEditContentPage(
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/masterspagetemplate/MastersPageTemplates.testcase:1756]: Should use Navigator.gotoBack();

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/masterspagetemplate/MastersPageTemplates.testcase [hunk 8]
```diff
@@ -1710,4 +1710,62 @@ definition {
...
+				pageName = "Test Page Name",
+				siteName = "Test Site Name");
+
+			PageEditor.gotoTab(tabName = "Page Design Options");
+
+			Click(locator1 = "PageEditor#MORE_LINK");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/masterspagetemplate/MastersPageTemplates.testcase:1762]: Should be
`AssertElementPresent( key_card = "${masterLayoutName}", locator1 = "Card#CARD_SELECTED");`

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/masterspagetemplate/MastersPageTemplates.testcase [hunk 9]
```diff
@@ -1710,4 +1710,62 @@ definition {
...
+			Navigator.gotoNavTab(navTab = "SEO");
+
+			PortletEntry.inputText(
+				fieldLabel = "HTML Title",
+				text = "Page HTML Title");
+
+			Button.clickSave();
+		}
+
+		task ("Assert in the Master tab if the Custom Masters Page Template is selected") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/masterspagetemplate/MastersPageTemplates.testcase:1755]: The description is different with steps.

**Comment by Tim-Cao** [comment]
Hi @ThalitaCeli, the branch number and PR summry should be automation ticket number while the number in description should be resolved ticket number.

**Comment by ThalitaCeli** [comment]
Done. 
If there are any other changes I'm here, ok?
Thank you!

**Comment by Tim-Cao** [comment]
See changes https://github.com/liferay-echo/liferay-portal/compare/5267b0e92c92c8a54d3425a7c7cf1bd9c10ce99b...65ef32a4b1916a37d85a9ec0e265a6f61d60e44b
https://github.com/liferay-echo/liferay-portal/pull/4924

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**